### PR TITLE
OCM-8387 | Remove `NonHCPCluster` and `NonClassicCluster` labels

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/OCM-8387

I set `!Exclude` as default label filter to avoid repetition in PR jobs

### CODE REVIEW

File: ci-operator/config/openshift/rosa/openshift-rosa-master__e2e-presubmits.yaml [hunk 1]
```diff
@@ -43,8 +43,6 @@ tests:
   optional: true
   steps:
     cluster_profile: aws-sd-qe
-    env:
-      TEST_LABEL_FILTERS: '!NonHCPCluster&&!Exclude'
```
Comment (xueli181114) [inline ci-operator/config/openshift/rosa/openshift-rosa-master__e2e-presubmits.yaml:47]: Why `!Exclude` also removed?

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-hcp pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-sts

**Comment by radtriste** [comment]
@kuiwang02 @bandrade Can I get an approval here ?

**Comment by yasun1** [comment]
/lgtm
@liangxia Could you help review the changes on openshift-tests-private? Thanks.

**Pull request merged**